mavisp/modules.py

```diff
@@ -111,21 +110,20 @@ def ingest(self, mutations):
         method = tmp[0]
 
         tmp = os.listdir(os.path.join(self.data_dir, self.module_dir, f'{structure_ID}_{residue_range}', method))
-        if len(tmp) != 1:
-            raise MAVISpMultipleError(warning=warnings,
-                                      critical=[MAVISpCriticalError(this_error)])
-        model = tmp[0]
+        results = []
```

Comment:
for this part of the code, I think the logic should be slightly different. We now can have cases like:

```
50frames/fold
4clusters/rosetta_...
```

```
4clusters/foldx
4clusters/rosetta...
```

which are fine, but also cases like:

```
50frames/rosetta...
4clusters/rosetta...
```

which are not fine. Could you refactor this code so that it considers this? i.e. we can have only one directory for every method (`foldx`, `rosetta...`)